src/OpenTelemetry.Instrumentation.Http/HttpClientInstrumentationOptions.cs

```diff
@@ -46,14 +47,13 @@ public class HttpClientInstrumentationOptions
 
         internal bool EventFilter(string activityName, object arg1)
         {
-            if (TryParseHttpRequestMessage(activityName, arg1, out HttpRequestMessage requestMessage))
-            {
-                return this.FilterFunc?.Invoke(requestMessage) ?? true;
-            }
-
-            return true;
+            return
```

Comment:
Let's put [MethodImpl(MethodImplOptions.AggressiveInlining)] on EventFilter too? My thinking is: When people aren't using the filter we should try to get as close to 0 perf hit as possible.